templates/shared_services/cyclecloud/terraform/locals.tf

```diff
@@ -1,6 +1,8 @@
 locals {
   core_resource_group_name = "rg-${var.tre_id}"
   core_vnet                = "vnet-${var.tre_id}"
+  allowed_fqdns            = "management.azure.com"
```

Comment:
Should be a service tag for this. AzureResourceManager or something.

Reply:
ill have a look